README.rdoc

```diff
@@ -1,5 +1,7 @@
 = test_linker
 
+This fork fixes test_linker to make it work with at least TestLink v1.9.13. <mipmip>
```

Comment:
This line makes sense in the context of your fork, but doesn't for the merge.  Can you remove?